### PR TITLE
MMDP-562: List stakeholders by location

### PR DESCRIPTION
**Pull request by fgkinus**
**What does this PR do?**

This PR adds functionality to search/list stakeholders in a certain location 

**description of Task to be completed?**
 Given a location name where location is `country`, `state`, `lga`,`community`, this endpoint will list all stakeholders with a footprint in the location including the count of stakeholders in this location

**How should this be manually tested?**
- git fetch `ft-list-stakeholders-by-location-MMDP-562`
- run `yarn run dev-server`
- Open postman and send a get request to `http://0.0.0.0:3000/api/v1/location?<query>`
  where `query` equals to either:
 ```
country = <value>
state = <value>
lga = <value>
community = <value>
```

**What are the relevant Jira issues?**

[MMDP-562](https://viisaus.atlassian.net/browse/MMDP-562)

**Screenshots (if appropriate)**

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "axios": "^0.18.0",
     "babel-polyfill": "^6.26.0",
     "babel-preset-react-app": "^7.0.0",
+    "bcrypt": "^3.0.5",
```
Comment (shemogumbe) [inline package.json]: Remove this, ensure you rebase all your branches with develop, bcrypt was replaced with bcrytpjs for compatibility issues

File: server/routes/api/resources/stakeholders/listStakeholdersByLocation.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+const searchBeneficiaryServiceCommunity = async (query) => {
+  let results = [];
+  const Model = keystone.list('BeneficiaryServiceCommunity');
+  let uniqueBeneficiaryServices = [];
+
+  // get all distinct entries in the table
+  await Model.model
+    .find(query)
+    .distinct('beneficiaryServiceId')
+    .then((docs) => {
```
Comment (shemogumbe) [inline server/routes/api/resources/stakeholders/listStakeholdersByLocation.js]: LGTM

**Comment by eltNEG** [comment]
@fgkinus @missvicki, Nice work. It works as expected. Can you kindly add pagination as suggested by @shemogumbe during one of our meetings?

**Comment by missvicki** [comment]
> @fgkinus @missvicki, Nice work. It works as expected. Can you kindly add pagination as suggested by @shemogumbe during one of our meetings?

Thank you @eltNEG, we had a discussion on the pagination and shall proceed as discussed.

**Pull request merged**